packages/reboundx/test_reboundx.py

```diff
@@ -0,0 +1,28 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(
+    packages=[
+        "rebound",
```

Comment:
You need:
```suggestion
        "rebound",
        "reboundx",        
```
Strangely enough the CI checks are okay even though this test failed...

Reply:
> Strangely enough the CI checks are okay even though this test failed...

Yeah, that is a bug in our test (#2915)